tests/functional/cylc-set/06-parentless/flow.cylc

```diff
@@ -19,4 +15,6 @@
             @wall_clock => x => b
         """
 [runtime]
-    [[a, b, x]]
+    [[a]]
```

Comment:
Why this refactor? Won't x run without the set?

Reply:
7a9d25305027cdc6decae900d375029a5af4fb28 - De-flakify test (for MacOS CI)